tests/e2e/test_plans.py

```diff
@@ -0,0 +1,36 @@
+from conftest import test_data_billing_plan, test_data_update_plan, test_update_tenant_billing_plan
+
+
+def test_list_billing_plans(client):
+    # Test create billing plan
+    resp = client.create_billing_plan(billing_plan_definition=test_data_billing_plan())
```

Comment:
Since I guess only Super User (_mm or whoever is super user) can only access these APIs right? So if that is the case then we should new client login instance for mm tenant in e2e/conftest.py and add new variables in tests/.env which hold the credentials for Super User login and update the same in readme.md

Reply:
Added new fixture in conftest with mm tenant. (Normal users can access GET requests only for these endpoints)